types.d.ts

```diff
@@ -134,5 +136,7 @@ export class SparseMerkleTree {
     update(root: Buffer, kvpair: { key: Buffer, value: Buffer }[]): Promise<Buffer>;
     prove(root: Buffer, queries: Buffer[]): Promise<Proof>;
     verify(root: Buffer, queries: Buffer[], proof: Proof): Promise<boolean>;
+    verifyInclusionProof(root: Buffer, queries: Buffer[], proof: Proof): Promise<boolean>;
+    verifyNonInclusionProof(root: Buffer, queries: Buffer[], proof: Proof): Promise<boolean>;
```

Comment:
I think these are not added in the source file

Reply:
You mean which file?

Reply:
SparseMerkleTree class is defined in [here](https://github.com/LiskHQ/lisk-db/blob/f9784fa4d1e4081c6dc20ff8b2c4959ad7da634f/sparse_merkle_tree.js), and this is untouched?

Reply:
We already have those functions in that file.

Reply:
oh yea that's true 😮  i guess it has the same issues in this file as [this](https://github.com/LiskHQ/lisk-db/pull/118#discussion_r1271834394)

Reply:
Also I solved the return problem that you mentioned in state_db for them too.

Reply:
Done.